python/fledge/common/logger.py

```diff
@@ -64,6 +65,15 @@ def setup(logger_name: str = None,
     .. _logging.getLogger: https://docs.python.org/3/library/logging.html#logging.getLogger
     """
 
+    def _get_process_name():
```

Comment:
It's a shame we can't get access to the argument from here. At one point we talked of having a task/service base class the would be parsing the arguments amongst other things. At least we don't do this too frequently.